plugins/basic/python2.6libs/pythonrc.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+This file is loaded automatically by Houdini at startup.
+"""
+
+import inspect
+import os
+import sys
+
+def plugin_startup():
+
+    # use inspect to get the current file path since attempts to access
+    # __file__ result in a NameError.
+    current_file_path = os.path.abspath(
+        inspect.getsourcefile(lambda: 0)
+    )
+
+    # construct the path to the plugin root's folder, 2 folders above this file.
```

Comment:
isn't it just one level above this file? Maybe this could be made clearer by something like

```
current_dir_path  = os.path.dirname(current_file_path)
plugin_root_path = os.path.abspath(os.path.join(current_dir_path, ".."))
```